scripts/index.js

```diff
@@ -21,19 +21,6 @@ const datapackageJson = async (url, options={}) => {
   return body
 }
 
-const pageLoadTime = async (url) => {
```

Comment:
Why are we deleting this?

Reply:
I am not testing this param, it was implemented in the first place. I can uncomment it

Reply:
but why it was there?

Reply:
Well, we used this function to calculate loading time and record it into csv file. Then we changed logic not to write, just checking all params, after that it was left. I want to keep it in  future use cases.